Project.toml

```diff
@@ -37,7 +37,6 @@ PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
 PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
 RCall = "6f49c342-dc21-5d91-9882-a32aef131414"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
-DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
```

Comment:
I think removing this will also require removing the compat entry below, no? Basically revert 8518c79e4622f56e2270de50096c9bfe5f9438ef